code/argv_list.py

```diff
@@ -1,2 +1,4 @@
+#!/usr/bin/env python
 import sys
-print('sys.argv is', sys.argv)
+
+print(f'''sys.argv is {sys.argv}''')
```

Comment:
I wasn't familiar with f-strings before this PR, so I appreciate you bringing it up as something to consider using. I found the [PEP](https://www.python.org/dev/peps/pep-0498/) for it, but can you point me to where you saw this as being the recommended way of string formatting right now? I didn't see it mentioned as such in the [official docs](https://docs.python.org/3.9/library/string.html#format-string-syntax).

Reply:
This is actually mentioned in the PEP you provided, the section "Rationale" starts with the following:
 "This PEP is driven by the desire to have a simpler way to format strings in Python. The existing ways of formatting are either error prone, inflexible, or cumbersome."

Reply:
Looking at the lesson, we don't teach `format()` or formatting strings with `%` , though we do have a few challenges that use the latter between episodes 11 and 12. Since we don't teach these older ways or the f-string formatted string literals, I would suggest consistently using `print()` with multiple arguments as we are now. While knowing a particular way/syntax of formatting strings in Python can be of use to those continuing to code with Python past the workshop, it is not something we necessarily need to go into detail with in trying to cover general programming concepts in the workshop content. If others think we should teach string formatting more in depth we can look at how we want to go about it. @annefou @maxim-belkin any thoughts on string formatting?